sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -778,8 +855,11 @@ def test_warm_start_oob_switch(Cls):
     est.fit(X, y)
 
     assert_array_equal(est.oob_improvement_[:100], np.zeros(100))
+    assert_array_equal(est.oob_scores_[:100], np.zeros(100))
```

Comment:
We don't have any test that checks `oob_score_` and the equality with `oob_scores_[-1]`.
We should be adding this in the different tests as well.